src/query/frontend/ast/ast.hpp

```diff
@@ -2253,6 +2253,7 @@ class CallProcedure : public memgraph::query::Clause {
   memgraph::query::Expression *memory_limit_{nullptr};
   size_t memory_scale_{1024U};
   bool is_write_;
+  bool is_util_validate_procedure_{false};
```

Comment:
Shouldn't this be more generic, meaning, does a procedure `YIELD` some values or no? I think the implementation can be more generic? 🤔

Reply:
Agree, better to just have a flag telling us that the CALL is without YIELD. Later on we can throw if the procedure called is con compatible with a CALL without YIELD.